### PR TITLE
Updates css for mkdocs to add border.

### LINKED ISSUE #5635: Docs: Add Border on Tabbed Information
### Proposed Changes

Add a border to distinguished tabbed documentation from non-tabbed documentation in the process.

### Justification

Aesthetics

### PR DESCRIPTION
**Pull request by jvanderaa**
# Closes #5635 
# What's Changed

Updated CSS from a recommendation about adding borders around tabbed interfaces on the mkdocs.

# Screenshots

**Before**
Screenshot 2024-04-25 at 21 01 33

**After**
Screenshot 2024-04-25 at 21 01 40

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example App Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

File: nautobot/docs/assets/extra.css [hunk 1]
```diff
@@ -137,3 +137,12 @@ img.copyright-logo {
...
 }
+
+.md-typeset .tabbed-set {
+    border: 0.5px solid gray;
+}
+
+.md-typeset .tabbed-content {
+    padding-left: 1rem;
+    padding-right: 1rem;
+}
```
Comment (glennmatthews) [inline nautobot/docs/assets/extra.css:148]: Out of curiosity, does it look okay/better without this padding?

File: changes/5635.housekeeping [hunk 2]
```diff
@@ -0,0 +1 @@
+Added borders to tabbed sections of mkdocs.
```
Comment (glennmatthews) [inline changes/5635.housekeeping:1]: Possibly change to a `.documentation` rather than a `.housekeeping`? Not critical.

**Review by glennmatthews — APPROVED** [review]

**Comment by jvanderaa** [comment]
Without the padding:

Screenshot 2024-04-26 at 11 51 50

**Comment by jvanderaa** [comment]
I think without the padding looks better. I will update the PR.

File: nautobot/docs/assets/extra.css [hunk 3]
```diff
@@ -137,3 +137,12 @@ img.copyright-logo {
...
 }
+
+.md-typeset .tabbed-set {
+    border: 0.5px solid gray;
+}
+
+/* .md-typeset .tabbed-content {
+    padding-left: 1rem;
+    padding-right: 1rem;
+} */
```
Comment (glennmatthews) [inline nautobot/docs/assets/extra.css:148]:
```suggestion
```

**Review by glennmatthews — APPROVED** [review]

File: nautobot/docs/assets/extra.css (hunk 3 above)
Comment (jvanderaa) [reply nautobot/docs/assets/extra.css]: I just got too excited about the look and feel!!!

Comment (jvanderaa) [reply nautobot/docs/assets/extra.css]: And then jumping to meetings.

**Pull request merged**